modules/xfem/test/tests/moving_interface/verification/tests

```diff
@@ -0,0 +1,90 @@
+[Tests]
+  [./1D_xy_homog1mat]
+    type = Exodiff
+    input = 1D_xy_homog1mat.i
+    exodiff = '1D_xy_homog1mat_out.e 1D_xy_homog1mat_out.e-s002 1D_xy_homog1mat_out.e-s003 1D_xy_homog1mat_out.e-s004 1D_xy_homog1mat_out.e-s005'
+    map = false
+    unique_id = true
+    allow_test_objects = True
+    requirement = 'The LevelSetCutUserObject class shall be able to be used to verify that the XFEM module is capable of accurately solving 1D, xy problems with homogeneous material properties with a moving interface developed via the Method of Manufactured Solutions.'
+    design = 'LevelSetCutUserObject.md'
+    issues = '#12833'
+  [../]
+  [./1D_xy_lsdep1mat]
+    type = Exodiff
+    input = 1D_xy_lsdep1mat.i
+    exodiff = '1D_xy_lsdep1mat_out.e 1D_xy_lsdep1mat_out.e-s002 1D_xy_lsdep1mat_out.e-s003 1D_xy_lsdep1mat_out.e-s004 1D_xy_lsdep1mat_out.e-s005'
+    map = false
+    unique_id = true
+    allow_test_objects = True
+    requirement = 'The LevelSetCutUserObject class shall be able to be used to verify that the XFEM module is capable of accurately solving 1D, xy problems with material properties and a moving interface that are dependent on a user prescribed level set function developed via the Method of Manufactured Solutions.'
```

Comment:
Maybe clarify that there's a jump in the material property over the interface.

Reply:
There's not really a jump in the material property over the interface; the thermal conductivity is dependent on the level set function's value (so it's continuous). This is addressed in the requirement: "...with material properties and a moving interface that are dependent on a user prescribed level set function..."